Fixes #5745 - Create Lakenheath (EGUL) SMR

Liaely opened the pull request:
Fixes #5745 

# Summary of changes

Created SMR for Lakenheath (EGUL).

# Screenshots

### Overall view:

![image](https://github.com/user-attachments/assets/f860fdfb-7aca-4bdb-a038-8edfe9f491cc)

### Aprons A & C:

![image](https://github.com/user-attachments/assets/b8862831-0c91-4de6-9671-c47ca3b4fc39)
![image](https://github.com/user-attachments/assets/85520f64-b375-4fae-bc83-7b625c844037)

BenWalker01 reviewed (changes requested):
Looks nice, but far too detailed. Ideally if you can get it under ~ 3000 lines. Runway markings / buidlings and roads can be removed. Probably also the centerlines of the taxiway

Liaely commented:
I've removed the centrelines, and updated the aprons and EOR areas to stands.
Yes apron V is weird like that.
I'll get around to reviewing the buildings and will probably remove a few small buildings, a bit later.

Liaely commented:
I've reduced the line count significantly to ~5k.
I can reduce the count further, but to do so at this point would in my opinion significantly reduce the quality, especially if it were to be applied consistently across the SMR.

Liaely commented:
@BenWalker01 done!
